Differentiate cloud provider credential from a workload identity (#78)
ietf-wg-wimse/draft-ietf-wimse-workload-identity-practices: draft-ietf-wimse-workload-identity-practices.md

     known, but local-only location such as 169.254.169.254.

When the workload needs to access a resource within the cloud (e.g., located in
the same security boundary; protected by the same issuer as the workload
identity):

* A) The workload directly accesses the protected resource with the credential
  issued in Step 1.

When the workload needs to access a resource outside of the cloud (e.g.,
different cloud; same cloud, but different security boundary):

* B1) The workload uses the cloud-issued credential to federate to the Secure Token
      Service of the other cloud/account. The STS validates the credential and
      issues a new credential (e.g., an access token) to the workload.

* B2) Using the credential issued in step B1, the workload can access the
      resource outside, assuming the credential has the necessary permissions.

It is important to distinguish the credential obtained from the Instance Metadata
Service from a workload identity document commonly used in attestation systems.
A workload identity document typically represents attestation evidence
that is evaluated by a relying party or attestation service. In contrast,

> the credential issued by the metadata service is already the result of such

Reviewer: ```suggestion
some credentials issued by the metadata service are already the result of such
```

Reviewer: AWS offers both, IAM creds and instance identity document (signed attestation results)
Azure too, Managed Identity creds and attested documents (not sure how wide of a support the ladder has)
GCP I'm not super familiar with.

> attestation and is intended to be directly consumed by relying services for

Reviewer: ```suggestion
attestation and are intended to be directly consumed by relying services for
```

> attestation and is intended to be directly consumed by relying services for

Author: ```suggestion
attestation and are intended to be directly consumed by relying services for
```
